ds_judgements_public_ui/templates/pages/accessibility_statement.html

```diff
@@ -41,7 +41,7 @@ <h3 class="anchor-links__header">On this page</h3>
       <div class="govuk-grid-column-two-thirds">
         <p class="date-time-info-page">
           <small>Last updated on
-            <time datetime="20-14-2024">20th August 2024</time>
+            <time datetime="03-10-2024">3 October 2024</time>
```

Comment:
@Terry-Price the th has already been removed see above change in code and image below 
![removed-th](https://github.com/user-attachments/assets/ce989e24-0bbb-4dc3-bf00-7d5b03463e75)

Reply:
Thanks @Donna-H